UI/pages/admin/dashboard.html

```diff
@@ -0,0 +1,180 @@
+            <div class="delete-options">
+              <button class="btn view-btn dismiss">Cancel</button>
+              <button class="btn delete-btn dismiss">Delete</button>
+            </div>
+          </div>
+        </div>
+        <table class="table fullwidth">
+          <thead>
+            <tr>
+              <th class="table-id">#</th>
+              <th class="table-name">Username</th>
+              <th class="table-title">Title</th>
+              <th class="table-start-date">Start Date</th>
+              <th class="table-start-end">End Date</th class="table-start-date">
+              <th class="table-status">Status</th class="table-start-date">
+              <th class="table-options center">Options</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>1</td>
+              <td>Digital Marketing Agency 1</td>
+              <td>Public Announcement Service 1</td>
+              <td>10/02/2020</td>
+              <td>30/03/2020</td>
+              <td>Pending</td>
+              <td class="options">
+                <a class="btn view-btn open-modal" href="#" onclick="openModal();">View</a>
+                <a class="btn delete-btn" href="#" onclick="deleteAnnouncement();">Delete</a>
+              </td>
+            </tr>
+            <tr>
+              <td>2</td>
+              <td>Digital Marketing Agency 2</td>
+              <td>Public Announcement Service 2</td>
+              <td>10/02/2020</td>
+              <td>30/03/2020</td>
+              <td>Pending</td>
+              <td class="options">
+                <a class="btn view-btn open-modal" href="#" onclick="openModal();">View</a>
+                <a class="btn delete-btn" href="#" onclick="deleteAnnouncement();">Delete</a>
+              </td>
+            </tr>
+            <tr>
+              <td>5</td>
+              <td>Digital Marketing Agency 5</td>
+              <td>Public Announcement Service 20</td>
+              <td>20/02/2020</td>
+              <td>15/03/2020</td>
+              <td>Active</td>
+              <td class="options">
+                <a class="btn view-btn open-modal" href="#" onclick="openModal();">View</a>
+                <a class="btn delete-btn" href="#" onclick="deleteAnnouncement();">Delete</a>
+              </td>
+            </tr>
+            <tr>
+              <td>2</td>
+              <td>Digital Marketing Agency 2</td>
+              <td>Public Announcement Service 12</td>
+              <td>1/02/2020</td>
+              <td>10/04/2020</td>
+              <td>Active</td>
+              <td class="options">
+                <a class="btn view-btn open-modal" href="#" onclick="openModal();">View</a>
+                <a class="btn delete-btn" href="#" onclick="deleteAnnouncement();">Delete</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+  <script type="text/javascript">
```

Comment:
I have left a comment about this on another PR: https://github.com/the22mastermind/announceIT/pull/10#discussion_r368296343